Dockerfile

```diff
@@ -41,6 +42,9 @@ RUN /bin/true \
             libboost-all-dev \
             libhdf5-dev \
             librabbitmq-dev \
+            rapidjson-dev \
```

Comment:
Why is rapidjson required now, or why was it not required before? Same question for libyaml and pybind11

Reply:
RapidJSON and YAML-cpp are now required because they were formerly included as submodules of Scarab, and have now been removed in favor of having them as external dependencies.  This simplified the submodule structure of these packages significantly and made cloning the repos simpler and faster.

Pybind11 is now required because of the change in structure of dripline-python.  Previously dl-py and dl-cpp were independent dripline implementations.  Now dl-cpp implements all of the core dripline functionality and dl-py wraps and extends dl-cpp with pybind11.